source/release-notes/1.8.txt

```diff
@@ -21,9 +21,91 @@ Release Notes for mongosync 1.8
 
 **Upcoming**
 
-New Features:
+Embedded Verifier
+~~~~~~~~~~~~~~~~~
+
+Starting in 1.8, ``mongosync`` includes an optional embedded
+verifier that performs a series of checks against the source and 
+destination clusters to verify a successful sync.
+
+For more information, see :ref:`c2c-embedded-verifier`.
+
+Other Notes
+~~~~~~~~~~~
+
+Updates to ``mongosync`` ``/progress`` API endpoint:
+
+- The ``/progress`` endpoint now returns a top-level boolean 
```

Comment:
(suggestion -- minimalism)
Since the lead in sentence says "Updates to mongosync /progress API endpoints, I think you can avoid repeating that in each bullet item.

Reply:
I have updated the bullet points to start with the verb (`returns`, `reports`, etc).